File: w09/d01/matthew_korporaal/mail_object/spec/mail_item_spec.rb
require_relative 'spec_helper'
require_relative '../mail_item'

describe MailItem do

  it "should have a ship_to_address" do
    mail = MailItem.new
    expect(mail.ship_to_address).to be
  end

  it "should have a stamp_value" do
    mail = MailItem.new
    expect(mail.stamp_value).to be
  end

end

File: w09/d03/jorge_colindres/mail_objects_app/spec/javascripts/mailItem_spec.js

describe('a mail item', function(){

  var item;
  beforeEach(function(){
    item = new MailItem('.99', '123 Main');
  });

  it('should have a stampValue', function(){
    expect(item.stampValue).toBeDefined();
    expect(item.stampValue).toBe('.99');
  });

  it('should have a shipToAddress', function(){
    expect(item.shipToAddress).toBeDefined();
    expect(item.shipToAddress).toBe('123 Main');
  });
});
